src/js/popup/popup.js

```diff
@@ -155,14 +155,13 @@ const serverStoragePanel = {
 
 async function choosePanel(numRemaining, panelId, premium, premiumSubdomainSet){
   const premiumPanelWrapper = document.querySelector(".premium-wrapper");
-  const premiumCountryAvailability = (await browser.storage.local.get("premiumCountries"))?.premiumCountries;
-
+  
   // Turned off label sync prompt for premium release
   // const shouldShowServerStoragePromptPanel = await serverStoragePanel.isRelevant();
   // if (shouldShowServerStoragePromptPanel) {
   //   serverStoragePanel.init(premium);
   // } else 
-  if (premium && premiumCountryAvailability?.premium_available_in_country === true) {
```

Comment:
@Vinnl Was there a specific use-case for limiting the premium panel to only Premium country-available users?

Reply:
tl;dr No, the country availability check can (and should) be removed.

Looking at the PR that introduced it (#246), I simply went by the name of the function I replaced (`premiumFeaturesAvailable`) and replaced that with this check. Of course, whether the Premium feature flag was enabled is actually something different than whether Premium is available in the user's current country, and in this case, if the user already has Premium even if they're in a different country where Premium is not available, it should still be shown.

Reply:
I just went over other instances of this check, but those refer to the upgrade prompts ("Get unlimited aliases"), so this is the only one that should be removed :)

Reply:
@Vinnl Thanks for the background/double-check!